lib/rake/circle.rake

```diff
@@ -128,6 +135,13 @@ namespace :circle do
     end
     close_sauce_connect if use_saucelabs || test_eyes?
     RakeUtils.system_stream_output 'sleep 10'
+
+    if GitUtils.changed_in_branch_or_local?(GitUtils.current_branch, ['dashboard/config/locales/*.en.yml'])
+      raise 'Unexpected change to dashboard/config/locales/ - Make sure you running seeding locally and include those changes in your branch.'
+    end
+    if GitUtils.changed_in_branch_or_local?(GitUtils.current_branch, ['dashboard/db/schema.rb'])
+      raise 'Unexpected change to schema.rb - Make sure you run your migration locally and push those changes into your branch.'
+    end
```

Comment:
nice helpful error messages, Dani! I was going to suggest turning this into an array people can add to, but I like how this gives a more useful error message in each case instead.

my one concern is that I might not know what to do next if I run migrate or seed and the problem does not go away. To address this, I would recommend including a diff of the affected file or directory, like we do here: https://github.com/code-dot-org/code-dot-org/blob/daaec080bb05d53e1292ade5708135fe16527249/lib/rake/circle.rake#L137-L141

for example, you should be able to add a line like one of these before you `raise`: 
```
RakeUtils.system_stream_output('git diff -- dashboard/db/schema.rb | cat')
RakeUtils.system_stream_output('git diff -- dashboard/config/locales | cat')
```